enterprise/dockerfiles/rbe-ubuntu22-04-workflows/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gcr.io/flame-public/rbe-ubuntu22-04@sha256:f068d42c574bff8e5c08620eb74d1cbda1c34c1e7f784252c5f5e675a0192409
+
+# Install bazelisk
+RUN curl -Lo /usr/local/bin/bazelisk https://github.com/bazelbuild/bazelisk/releases/download/v1.15.0/bazelisk-linux-amd64 && \
+    chmod +x /usr/local/bin/bazelisk
+
+# Pre-download/extract bazel so that Bazel can skip that work on first run,
+# at least for CI runs on the BB repo itself.
+RUN USE_BAZEL_VERSION=6.0.0 bazelisk version
```

Comment:
we could probably remove these - we provision our own version of bazelisk using the `ci_runner` and place it in PATH, and also Bazel 6.0.0 is out of date at this point (not to mention, this step is installing bazel only for the `root` user and workflows run by non-root as default now)

Reply:
Done.